Assets/MirageXR/Player/Scripts/Spatial/InfoScreenSpatialView.cs

```diff
@@ -43,9 +62,319 @@ public void Initialize(ActivityStep step)
             UpdateView(step); 
         }
         
+        private void Update()
+        {
```

Comment:
This will not work on the glasses - this should be a different, additional event?

Reply:
fixed